### PR TITLE
5.0.2 devel non doc pieces for deployment

### PR DESCRIPTION
**Pull request by szadam**
This change is [Reviewable](https://reviewable.io/reviews/tieredmemdb/tieredmemdb/27)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,20 +1,41 @@
...
+===========
 
-What is Redis?
---------------
+This project is a fork of [Redis](https://redis.io), adapted for systems
+with multiple memory tiers.  New memory technologies include memories that
+have very high bandwidth at the cost of latency, slower large capacity
+at lower cost, sharing memory among multiple machines in a rack, and so on.
 
+<<<<<<< HEAD
```
Comment (PatKamin) [inline README.md]: Resolve merge conflicts.

**Review by szadam — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/tieredmemdb/tieredmemdb/27)* status: 0 of 3 files reviewed, 1 unresolved discussion (waiting on @PatKamin)

___
*[`README.md` line 9 at r2](https://reviewable.io/reviews/tieredmemdb/tieredmemdb/27#-N2kTnQGKpCElOw0iOwL-r2-9:-N2koKJ38tGG8Ql2DW07:bb6un1j) ([raw file](https://github.com/tieredmemdb/tieredmemdb/blob/6865896022a127400c4c71d695fdf98184ba4a6e/README.md#L9)):*
<details><summary><i>Previously, PatKamin (Patryk Kaminski) wrote…</i></summary><blockquote>

Resolve merge conflicts.
</blockquote></details>

Done

File: README.md (hunk 1 above)
Comment (szadam) [reply README.md]: Done

**Review by PatKamin — APPROVED** [review]
> Reviewed 2 of 2 files at r1, 1 of 1 files at r3, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/tieredmemdb/tieredmemdb/27)* status: :shipit: complete! all files reviewed, all discussions resolved (waiting on @szadam)

**Pull request merged**